integration/presto/src/main/java/org/apache/carbondata/presto/CarbondataPageSource.java

```diff
@@ -47,25 +44,18 @@
 
   private static final LogService logger =
       LogServiceFactory.getLogService(CarbondataPageSource.class.getName());
-  private final List<Type> types;
+  private List<ColumnHandle> columnHandles;
   private boolean closed;
   private PrestoCarbonVectorizedRecordReader vectorReader;
-  private CarbonDictionaryDecodeReadSupport<Object[]> readSupport;
-  List<ColumnHandle> columnHandles;
   private long sizeOfData = 0;
-  private final StreamReader[] readers ;
   private int batchId;
   private long nanoStart;
   private long nanoEnd;
 
-  public CarbondataPageSource(CarbonDictionaryDecodeReadSupport readSupport,
-      PrestoCarbonVectorizedRecordReader vectorizedRecordReader,
-      List<ColumnHandle> columnHandles ) {
+  CarbondataPageSource(PrestoCarbonVectorizedRecordReader vectorizedRecordReader,
```

Comment:
no need of creating streamReader and dictionary read support here, as they are moved to  CarbonVectorBatch